cli/portforward_test.go

```diff
@@ -148,6 +159,12 @@ func TestPortForward(t *testing.T) {
 			cancel()
 			err = <-errC
 			require.ErrorIs(t, err, context.Canceled)
+
+			wuTick <- dbtime.Now()
```

Comment:
Consider also selecting on ctx in case consumer goes away (lets the test exit). (Flush too?)